services/command-router/src/main/java/org/eclipse/hono/commandrouter/impl/DelegatingCommandRouterAmqpEndpoint.java

```diff
@@ -268,6 +282,70 @@ protected Future<Message> processUnregisterCommandConsumer(final Message request
         return finishSpanOnFutureCompletion(span, resultFuture);
     }
 
+    /**
+     * Processes a <em>unregister command consumers</em> request message.
+     *
+     * @param request The request message.
+     * @param targetAddress The address the message is sent to.
+     * @param spanContext The span context representing the request to be processed.
+     * @return The response to send to the client via the event bus.
+     */
+    protected Future<Message> processUnregisterCommandConsumers(final Message request,
```

Comment:
can we rename this to e.g. *processUnregisterMultipleCommandConsumers* so that it is easier to distinguish from the existing *processUnregisterCommandConsumer* method? Maybe we should even rename the existing method to *processUnregisterSingleCommandConsumer* ...

Reply:
Agree, that having Single and Multiple as part of the name is easier to read.
What about the _CommandRouterAction_ enum? Does it make sense to have the same naming as part of the enum predefined constants?